data-prepper-api/src/test/java/com/amazon/dataprepper/model/buffer/AbstractBufferTest.java

```diff
@@ -64,8 +64,8 @@ public void testWriteMetrics() throws TimeoutException {
         Assert.assertTrue(
                 MetricsTestUtil.isBetween(
                         MetricsTestUtil.getMeasurementFromList(writeTimeMeasurements, Statistic.TOTAL_TIME).getValue(),
-                        0.5,
-                        0.6));
+                        0.4,
```

Comment:
Would `0.45` and `0.65` work here? Similar to what you did in `testWriteAllMetrics`?

Reply:
They should still work yes. I can tighten it to that.